src/main.rs

```diff
@@ -2,21 +2,17 @@
 #![no_main]
 
 use esp_backtrace as _;
-use esp_hal::{
-    clock::ClockControl, delay::Delay, peripherals::Peripherals, prelude::*, system::SystemControl,
-};
+use esp_hal::{delay::Delay, prelude::*};
 
 {% if alloc -%}
 {{ alloc_snippet }}
 {% endif -%}
 
 #[entry]
 fn main() -> ! {
-    let peripherals = Peripherals::take();
-    let system = SystemControl::new(peripherals.SYSTEM);
-
-    let clocks = ClockControl::max(system.clock_control).freeze();
-    let delay = Delay::new(&clocks);
+    #[allow(unused)]
```

Comment:
Don't love this, but unfortunately depending on which options the user selects during generation, this may or may not be used. Only way I could get CI green.